### PR TITLE
Add more features to pegout flow

### PR DESCRIPTION
**Pull request by lserra-iov**
Screenshot 2024-05-29 at 12 58 45

### CODE REVIEW

File: src/pegout/components/FlyoverRbtcInputAmount.vue [hunk 1]
```diff
@@ -98,23 +105,46 @@ export default defineComponent({
...
+          context.emit('get-quotes');
+        }
+      },
+    });
 
     const safeAmount = computed((): WeiBig => new WeiBig(rbtcAmount.value ?? '0', 'rbtc'));
 
     const amountErrorMessage = computed(() => {
-      const { minValue, maxValue } = minMaxValues.value;
+      const { minValue, maxValue } = pegoutConfiguration.value;
```
Comment (alexjavabraz) [inline src/pegout/components/FlyoverRbtcInputAmount.vue:147]: Please set default values inside the peginConfiguration class: min 0.005 and max 1

Comment (lserra-iov) [reply src/pegout/components/FlyoverRbtcInputAmount.vue:147]: You mean this value (https://github.com/rsksmart/2wp-app/blob/feature/pegout-flow-2/src/main.ts#L20)?

File: src/common/utils/utils.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export function getEstimatedFee(): Promise<SatoshiBig> {
       bridgeService.getQueuedPegoutsCount(),
     ])
       .then(([nextPegoutCost, pegoutQueueCount]) => {
-        const estimatedFee = nextPegoutCost / (pegoutQueueCount + 1);
+        const estimatedFee = nextPegoutCost / (pegoutQueueCount + 1n);
```
Comment (alexjavabraz) [inline src/common/utils/utils.ts:67]: ``
```suggestion
        const estimatedFee = (nextPegoutCost ? 1 : nextPegoutCost) / ((pegoutQueueCount ? 0 : pegoutQueueCount) + 1n);
```

**Review by alexjavabraz — CHANGES_REQUESTED** [review]

File: src/common/utils/utils.ts (hunk 2 above)
Comment (lserra-iov) [reply src/common/utils/utils.ts]: I get your point but `nextPegoutCost` should never be 0 according to [RSKIP385](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP385.md). If we want to be extra cautious it is not as simple as using a default value, we must do the calc.

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Pull request merged**